controller/participants/participantsController.js

```diff
@@ -27,26 +30,16 @@ router.use(async function (req, res, next) {
 
 
 router.get("/getFeed", async (req, res, next) => {
-  /* Check the req, if there are required paramaters missing, throw error.
```

Comment:
למה למחוק? הערה חשובה

Reply:
It was like a TODO comment. This is an obvious comment and not specific/helping at all

Reply:
Leave it as TODO so I would know that this is the part responsible for this stuff

Reply:
Reminder